test/requirements.txt

```diff
@@ -2,4 +2,5 @@ psycopg2
 MySQL-python
 PyGreSQL
 DBUtils
-PyMySQL
\ No newline at end of file
+PyMySQL
+MySQLConnector
```

Comment:
Please have a look at the following discussion on stackoverflow to see how to install it using pip.

http://stackoverflow.com/questions/22100757/can-not-get-mysql-connector-python-to-install-in-virtualenv